src/_layouts/default.erb

```diff
@@ -9,7 +9,7 @@
         <header
           class="contents lg:pointer-events-none lg:fixed lg:inset-0 lg:z-40 lg:flex"
           >
-          <div class="contents lg:pointer-events-auto lg:block lg:w-72 lg:overflow-y-auto lg:border-r lg:border-zinc-900/10 xl:w-80 lg:dark:border-white/10">
+          <div id="context-save" class="contents lg:pointer-events-auto lg:block lg:w-72 lg:overflow-y-auto lg:border-r lg:border-zinc-900/10 xl:w-80 lg:dark:border-white/10">
```

Comment:
please rename the id to something that provides context on what the element is, like `id="navigation-sidebar"`